doc/getting-started.md

```diff
@@ -19,7 +21,24 @@ This guide will help you get started writing contracts with CTL. Please also see
 
 ## Prerequisites
 
-You need to have set up a Purescript project using CTL as a dependency ([more details here](./ctl-as-dependency.md)). You will also need to become familiar with [CTL's runtime](./runtime.md) as its runtime services are required for executing virtually all contracts.
+### Setting up a new project
+
+The easiest way to create a new CTL project is to use our `ctl-scaffold` flake template. This lives in the CTL repo -- you can have a look [here](../templates/ctl-scaffold). It contains a simple, yet complete, flakes-based scaffolding project with example `outputs` for a CTL project, including its runtime.
+
+A new project can be initialized as follows:
+
+```
+$ mkdir new-project && cd new-project
+$ nix flake init -t github:Plutonomicon/cardano-transaction-lib
```

Comment:
Can we put these command to CI?

Reply:
What do you mean? Do you mean add a script to CI that attempts to initialize the template?

Reply:
Yes. Because otherwise it may become outdated.

Reply:
Unfortunately using Nix inside Nix is not trivial (in fact there is even an experimental feature to enable this) and we would also need network access to actually run `nix flake init`. Also, `nix flake init` just copies the files pointed to by the `path` attribute, it doesn't tell us anything meaningful about whether or not the template will build

Instead what we could perhaps do is try to run `build-spago-style` or something by copying the template files from `"${self}/templates/ctl-scaffold/...` into the build directory

Reply:
I'll think about this more and add an issue

Reply:
https://github.com/Plutonomicon/cardano-transaction-lib/issues/839